geomesa-web/geomesa-web-stats/src/main/scala/org/locationtech/geomesa/web/stats/GeoMesaStatsEndpoint.scala

```diff
@@ -112,35 +135,37 @@ class GeoMesaStatsEndpoint extends GeoMesaScalatraServlet with LazyLogging with
 
         jsonBoundsList.mkString("{", ", ", "}")
 
-      case _ => BadRequest(s"No registered layer called ${params("layer")}")
+      case _ => BadRequest(s"No registered layer called ${params(GeoMesaStatsEndpoint.LayerParam)}")
     }
   }
 
-  /**
-    * Retrieves a histogram of attributes.
-    * (only works against attributes which are cached in the stats table)
-    *
-    * params:
-    *   'attributes' - Comma separated attributes list to retrieve histograms for.
-    *                  If omitted, the command will create histograms for all attributes.
-    *   'bins'       - Number of bins as an integer
-    */
-  get("/:workspace/:layer/histogram") {
+  val getHistograms = (
+    apiOperation[String]("getHistogram")
+      summary "Gets histograms of attributes"
+      notes "Gets histograms of attributes from the stats table in Accumulo."
+      parameters (
+      pathParam[String]("workspace").description("GeoServer workspace."),
```

Comment:
[style] please indent the params an extra tab so it's more obvious it's a list
ditto in the other swagger definitions